src/main/java/uk/gov/hmcts/darts/common/entity/base/CreatedModifiedBaseEntity.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.darts.common.entity.base;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.MappedSuperclass;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import uk.gov.hmcts.darts.common.entity.UserAccountEntity;
+
+import java.time.OffsetDateTime;
+
+@MappedSuperclass
+@Getter
+@Setter
+public class CreatedModifiedBaseEntity {
```

Comment:
FYI Looks like Barry will be reinstating the `version integer not null` column soon, so we can pull that column from the `VersionedEntity` class to be our sensible base class. Now that the entities are becoming more uniformly presented.